huaweicloud/services/apig/data_source_huaweicloud_apig_instance_associated_ssl_certificates.go

```diff
@@ -0,0 +1,282 @@
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The ID of the SSL certificate.`,
+						},
+						"project_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The ID of the tenant project`,
+						},
+						"instance_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The ID of the dedicated instance to which the SSL certificates belong.`,
+						},
+						"name": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The name of the SSL certificate.`,
+						},
+						"type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The type of the SSL certificate.`,
+						},
+						"common_name": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The domain name of the SSL certificate.`,
+						},
+						"san": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: `The san extended domain of the SSL certificate.`,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+						"signature_algorithm": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The signature algorithm of the SSL certificate.`,
+						},
+						"algorithm_type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The algorithm type of the SSL certificate(RSA, ECC, SM2).`,
+						},
+						"is_has_trusted_root_ca": {
+							Type:        schema.TypeBool,
+							Computed:    true,
+							Description: `The certificate has trusted root certificate authority or not.`,
+						},
+						"not_after": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The expiration date of the SSL certificate.`,
+						},
+						"create_time": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The create time of the SSL certificate.`,
+						},
+						"update_time": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The update time of the SSL certificate.`,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func buildListInstanceAssociatedSSLCertificatesParams(d *schema.ResourceData) string {
+	res := ""
+	if v, ok := d.GetOk("name"); ok {
+		res = fmt.Sprintf("%s&name=%v", res, v)
+	}
+	if v, ok := d.GetOk("common_name"); ok {
+		res = fmt.Sprintf("%s&common_name=%v", res, v)
+	}
+	if v, ok := d.GetOk("signature_algorithm"); ok {
+		res = fmt.Sprintf("%s&signature_algorithm=%v", res, v)
+	}
+	if v, ok := d.GetOk("type"); ok {
+		res = fmt.Sprintf("%s&type=%v", res, v)
+	}
+	if v, ok := d.GetOk("algorithm_type"); ok {
+		res = fmt.Sprintf("%s&algorithm_type=%v", res, v)
+	}
+	return res
+}
+
+func queryInstanceAssociatedSSLCertificates(client *golangsdk.ServiceClient, d *schema.ResourceData) ([]interface{}, error) {
```

Comment:
```suggestion
func listInstanceAssociatedSSLCertificates(client *golangsdk.ServiceClient, d *schema.ResourceData) ([]interface{}, error) {
```
because of the parameter build function names `buildListXXX`

Reply:
done